apps/system/js/app_chrome.js

```diff
@@ -604,10 +637,34 @@
     });
   };
 
+  AppChrome.prototype._onBookmarksChange = function (evt) {
+    var targetUrl = evt.target.id;
+    if (this._currentURL === targetUrl) {
```

Comment:
Oh I see yeah, because we're pinning the site, the current URL might not match even though the site does.